client/src/components/Common/Button/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, ConfigProvider } from 'antd';
+import 'antd/dist/antd.css';
+
+const ButtonComponent = ({
+  Content,
+  disabled,
+  loading,
+  size,
```

Comment:
if you can ad comment that what we can add on this size prop, or we already have class.

Reply:
I don't use css-file to edit the size of the button , I use the antDesign-api to resize the button